sdk/servicebus/Azure.Messaging.ServiceBus/src/Diagnostics/MessagingDiagnosticOperation.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Core.Shared;
+
+internal readonly partial struct MessagingDiagnosticOperation
+{
+    public static MessagingDiagnosticOperation RenewMessageLock = new("renew_message_lock");
```

Comment:
based on offline discussion, I'm suggesting to remove all but `settle` for now - we might return them later on. 

If you came up with a good scenario for them that can help users or us to debug problems, I'm totally on board to add them now.

Reply:
I removed all of the other settlement methods, but things like renew message lock are not settlement. Should we just leave these blank?